pythonforandroid/build.py

```diff
@@ -615,7 +626,9 @@ def run_pymodules_install(ctx, modules):
 
     venv = sh.Command(ctx.virtualenv)
     with current_directory(join(ctx.build_dir)):
-        shprint(venv, '--python=python2.7', 'venv')
+        shprint(venv,
+                '--python=python{}'.format(ctx.python_recipe.major_minor_version_string()),
```

Comment:
`major_minor_version_string` is a property, not a method, so we should remove the brackets